MapboxNavigation/FeedbackItem.swift

```diff
@@ -7,7 +7,8 @@ extension UIImage {
     }
 }
 
-struct FeedbackItem {
+@objc(MBFeedbackItem)
+public class FeedbackItem: NSObject {
     var title: String
     var image: UIImage
     var feedbackType: FeedbackType
```

Comment:
Do any of these properties need to be public? What about the initializer?

Reply:
I think the initializer should be enough, no?

Reply:
The initializer is now public, but the arguments passed into the initializer are thereafter inaccessible to client code, making the feedback item something of a dropbox. I think we should make these three properties public for consistency.